web/controllers/v1/group_membership_controller.ex

```diff
@@ -18,6 +18,13 @@ defmodule Cog.V1.GroupMembershipController do
     render(conn, "show.json", group: group)
   end
 
+  def manage_group_users(conn, %{"users" => user_spec}=params) do
+    params
+    |> Map.put("members", %{"users" => user_spec})
+    |> Map.delete("users")
+    |> fn(param) -> manage_membership(conn, param) end.()
```

Comment:
Not really, it's just another instance of me trying to make the minimum possible changes so that I could add functionality back later if we change our minds. In this case I'm happy to just rename manage_group_users though since it's pretty small change.